app/javascript/react/screens/App/Overview/screens/PlanWizard/components/PlanWizardInstancePropertiesStep/components/PlanWizardInstancePropertiesStepTable.js

```diff
@@ -170,6 +202,29 @@ class PlanWizardInstancePropertiesStepTable extends React.Component {
       strategy: sort.strategies.byProperty
     });
 
+    const flavorColumnPopoverContent = (
+      <Popover id="v2v-osp-flavor-info" className="popover-with-max-width">
+        <p>
+          {__('In OpenStack, flavors define the compute, memory, and storage capacity of nova computing instances.')}
+        </p>
+        <p>
+          {__(
+            'A flavor name appended with an * indicates that the flavor has CPU or memory capacity smaller than the source VM. Selecting these flavors might cause problems when the VM is migrated.'
```

Comment:
Can you get the entire gettext expression in a single line and then add the `// prettier-ignore` comment?